mage_ai/streaming/sources/kafka.py

```diff
@@ -256,7 +256,8 @@ def batch_read(self, handler: Callable):
                 handler(message_values)
 
     def test_connection(self):
-        return True
+        self.consumer._client.check_version(timeout=5)
+        self._print('Test connectino successfully.')
```

Comment:
@wangxiaoyou1993 Mate, was using this, you misspelled this one. Need a hand to create a PR?

Reply:
oh, yeah. thanks for catching the typo. feel free to open a PR for it.